src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1716,33 +1716,29 @@
   <Node Name="PropertyPatternSyntax" Base="PatternSyntax">
     <Kind Name="PropertyPattern"/>
     <Field Name="Type" Type="TypeSyntax" />
-    <Field Name="PatternList" Type="SubPropertyPatternListSyntax" />
-  </Node>
-  <Node Name="SubPropertyPatternListSyntax" Base="CSharpSyntaxNode">
-    <Kind Name="SubPropertyPatternList"/>
     <Field Name="OpenBraceToken" Type="SyntaxToken">
       <Kind Name="OpenBraceToken"/>
       <PropertyComment>
-        <summary>SyntaxToken representing open brace.</summary>
+        <summary>SyntaxToken representing the open brace.</summary>
+      </PropertyComment>
+    </Field>
+    <Field Name="Expressions" Type="SeparatedSyntaxList&lt;ExpressionSyntax&gt;">
```

Comment:
We model this after object initializers. The contents can be a IsPatternExpression, but we use a general Expression so the parser can be more relaxed. We do not take advantage of that today, but I expect to improve the parser (for error recovery) and then make the binder more relaxed too.